codesy/base/models.py

```diff
@@ -1,26 +1,43 @@
 import requests
+import stripe
 
 from django.conf import settings
 from django.contrib.auth.models import AbstractUser
 from django.db import models
-from django.db.models.signals import post_save
+from django.db.models.signals import post_save, pre_save
 from django.dispatch import receiver
 
 from allauth.account.signals import user_signed_up
 from rest_framework.authtoken.models import Token
 
 
 EMAIL_URL = 'https://api.github.com/user/emails'
+stripe.api_key = settings.STRIPE_SECRET_KEY
 
 
 class User(AbstractUser):
-    stripe_card_token = models.CharField(max_length=100, blank=True)
-    # TODO: remove/replace User.balanced_bank_account_href
-    balanced_bank_account_href = models.CharField(max_length=100, blank=True)
-
+    stripe_account_token = models.CharField(max_length=100, blank=True)
     USERNAME_FIELD = 'username'
 
 
+@receiver(pre_save, sender=settings.AUTH_USER_MODEL)
+def replace_cc_token_with_account_token(sender, instance, **kwargs):
```

Comment:
This should be moved to separate endpoints so we aren't using the :sparkles: of temporarily storing the one-time CC token in the `stripe_account_token` field.